jabbercat/models.py

```diff
@@ -817,12 +814,12 @@ def filterAcceptsRow(self,
             # filter inbound subscription requests
             return False
 
-        if self._filter_by_text is not None:
-            if (not self._contains_normalized(self._filter_by_text,
-                                              str(item.address)) and
-                    not self._contains_normalized(self._filter_by_text,
-                                                  item.label)):
-                return False
+        if self._filter_by_text and self._filter_by_text:
```

Comment:
Huh? What is that about? Isn’t that equivalent to ``if self._filter_by_text:``?

Reply:
That, is a brainfart :)